src/components/Select/Select.tsx

```diff
@@ -242,6 +254,9 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
                 onKeyDown={handleControlKeyDown}
                 renderControl={renderControl}
                 value={value}
+                popupId={`select-popup-${selectId}`}
+                selectId={`select-${selectId}`}
+                activeIndex={activeIndex}
```

Comment:
I think we should add this properties to custom controls [here](https://github.com/gravity-ui/uikit/blob/d51039bb0fd4129ac511681e7fa23f051d1c3397/src/components/Select/components/SelectControl/SelectControl.tsx#L114)